mtags/src/main/scala-3/scala/meta/internal/pc/completions/MatchCaseCompletions.scala

```diff
@@ -311,13 +330,13 @@ object CaseKeywordCompletion:
     completions
   end matchContribute
 
-  private def sortSubclasses(
+  private def sortSubclasses[A](
       tpe: Type,
-      syms: List[Symbol],
+      syms: List[(Symbol, A)],
```

Comment:
```suggestion
      syms: List[(Symbol, A)],
```
is A needed? Could we instead have a stable class here?